### PR TITLE
NMS-9932: Fix wrong message displayed on AngularJS based tables.

### PR DESCRIPTION
**Pull request by cgorantla**
If resources are waiting to be loaded specify that instead of showing zero resources which could be misleading

* JIRA: http://issues.opennms.org/browse/NMS-9932

Our [continuous integration system](http://bamboo.internal.opennms.com:8085) will test and verify your changes.

### CODE REVIEW

**Comment by agalue** [comment]
Looks good to me. Looking at the code, I found a problem with my original implementation that I think it would be good to fix it as part of the solution.

As you can see, the Angular code is not handling errors after retrieving the information through ReST. The idea would be add a "catch" to the returned Promise and set "loaded" to be "false"; for example:

```JavaScript
 $http.get('rest/resources?depth=0').success(function(data) {
    $scope.loaded = true;
    $scope.hasResources = data.resource.length > 0;
    $scope.resources = data.resource;
    $scope.filteredResources = $scope.resources;
    $scope.update();
  }).error(function(error,status) {
    $scope.loaded = true;
    console.error("There was a problem retrieving the resources through ReST").
  });
```
That way, we nicely handle the case where there was a problem and then show that the table is empty. There are better ways to handle errors (which is how the requisitions UI behaves), but that should be enough for this use case.

Makes sense ?

**Comment by cgorantla** [comment]
Sure, I will fix that

File: core/web-assets/src/main/assets/js/apps/onms-resources/index.js [hunk 1]
```diff
@@ -51,6 +52,7 @@ angular.module('onms-resources', [
   };
 
   $http.get('rest/resources?depth=0').success(function(data) {
+    $scope.loaded = true;
```
Comment (agalue) [inline core/web-assets/src/main/assets/js/apps/onms-resources/index.js]: It seems like you've missed the error handler in this case (to set `$scope.loaded=false`).

Comment (cgorantla) [reply core/web-assets/src/main/assets/js/apps/onms-resources/index.js]: Thanks @agalue for catching, added it

**Comment by agalue** [comment]
Excellent, thanks! Once Bamboo goes green, feel free to merge it.

**Pull request merged**